src/ucm/bistro/bistro.c

```diff
@@ -48,23 +48,28 @@ static ucs_status_t ucm_bistro_protect(void *addr, size_t len, int prot)
     return UCS_OK;
 }
 
-void ucm_bistro_apply_locking(void *dst, const ucm_bistro_lock_t *bytes)
+static double ucm_bistro_get_time_msec()
```

Comment:
maybe move it to ucm/util/sys.{c,h} as "ucm_get_time"

Reply:
fixed, now using second units